app/src/main/java/org/mozilla/fenix/tabstray/browser/TabSorter.kt

```diff
@@ -23,16 +21,12 @@ class TabSorter(
     private val settings: Settings,
     private val tabsTrayStore: TabsTrayStore? = null
 ) : TabsTray {
-    private val groupsSet = mutableSetOf<String>()
 
     override fun updateTabs(tabs: List<TabSessionState>, tabPartition: TabPartition?, selectedTabId: String?) {
```

Comment:
`updateTabs` should be updated in the `TabsTray` interface to not have `TabPartition` anymore.

Reply:
This is an AC change, so I was debating whether it made sense to wait to remove it until search groups gets overhauled in the future.

Reply:
I see.  That's good thinking.  Maybe we should raise an issue in A-C to track this work?  In case we don't need it in the future?

Reply:
That sounds good to me. I'll post it here when I make it.

Reply:
Actually I think you're right.  We probably should keep the interface intact since it's related to the `tabPartition` work in the future.  Please ignore my comment here.  Thanks